doc_classes/BTSubtree.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		BTSubtree instantiates a [BehaviorTree] and includes its root task as a child during initialization, while also creating a new [Blackboard] scope.
 		Returns the status of the subtree's execution.
+		Note: BTSubTree is designed as a simpler loader, and does not support updating `subtree` at runtime. A custom subtree decorator is better suited and somewhat trivial to implement (https://github.com/limbonaut/limboai/issues/94#issuecomment-2068833610).
```

Comment:
You can use `[url=https://example.com]Example[/url]`. 
And also, members are referred like this: `[member subtree]` (instead of back quotes).

Reply:
It's not part of schema, I think it's substitutions that documentation parser uses on top of the xml, but I'm not 100% sure.

Reply:
Thanks!